### PR TITLE
Discovery v15 manual

### PR DESCRIPTION
**Pull request by lzap**
This might or might not conflict with

https://github.com/theforeman/theforeman.org/pull/1256

Just merge what comes first and I will rebase.

### CODE REVIEW

File: plugins/foreman_discovery/nightly/index.md [hunk 1]
```diff
@@ -223,7 +223,9 @@ on Fedora and Red Hat systems or
 
 on Debian and Ubuntu systems.
 
-Restart of Foreman is needed when installing the package separately.
+Restart of Foreman is needed when not using the installer:
+
+    # systemctl restart httpd
```
Comment (mmoll) [inline plugins/foreman_discovery/nightly/index.md:228]: On Debian/Ubuntu the service is called `apache2`

Comment (lzap) [reply plugins/foreman_discovery/nightly/index.md:228]: Oh right thanks! :-)

**Comment by lzap** [comment]
Done.

**Comment by mmoll** [comment]
merged, díky @lzap!

**Pull request merged**